public/app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistory.tsx

```diff
@@ -208,7 +208,10 @@ function AlertRuleName({ labels, ruleUID }: { labels: Record<string, string>; ru
   }
   return (
     <Tooltip content={alertRuleName ?? ''}>
-      <a href={`/alerting/${GRAFANA_DATASOURCE_NAME}/${ruleUID}/view`} className={styles.alertName}>
+      <a
+        href={`/alerting/${GRAFANA_RULES_SOURCE_NAME}/${ruleUID}/view?returnTo=${encodeURIComponent('/alerting/history')}`}
```

Comment:
Added a `returnTo` here so you end up back at history page

Reply:
💯